java/src/jmri/jmrit/display/layoutEditor/LayoutTurnout.java

```diff
@@ -1542,11 +1542,9 @@ public void setCoordsCenter(Point2D p) {
     }
 
     private void reCalculateCenter() {
-        double centreX = (pointC.getX() - pointA.getX()) / 2;
-        double centreY = (pointC.getY() - pointA.getY()) / 2;
-        centreX = pointA.getX() + centreX;
```

Comment:
Was modification of the `reCalculateCenter` method supposed to be part of this pull request?